account_invoice_triple_discount/views/account_move.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="invoice_triple_discount_form_view" model="ir.ui.view">
+        <field name="name">account.invoice.triple.discount.form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='invoice_line_ids']//tree//field[@name='discount']"
+                position="after"
+            >
+                <field name="discount2" optional="hide" string="Disc.2%" />
```

Comment:
@ramiadavid If you change the label, you need to update the translation files ..

Reply:
@lmignon This should be done once this is merged through weblate.

Reply:
@rousseldenis In this case, the source term is changed. That doesn't mean that we want to loose all the translations of the previous source term. That's why the translation files must be updated also replace the previous label by the new label for the source term.

Reply:
@rousseldenis Updated translations for translated langs